src/actor/ActorTemplate.js

```diff
@@ -205,34 +255,42 @@ exports = function(inherits) {
     }
 
     /**
-      * Set {@link Actor#vx} and {@link Actor#vy} to aim for the specified point, with the specified speed.
-      * @func Actor#headToward
-      * @arg {number} x
-      * @arg {number} y
-      * @arg {number} speed
-      */
+     * Set vx and vy to aim for the specified point, with the specified speed.
+     * @method ActorTemplate#headToward
```

Comment:
This fn should probably me `moveToward` ... the word `head` can mean different things.